cmd/workspace.go

```diff
@@ -198,6 +203,85 @@ func newWorkspaceUserListCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+/*
+	Workspace teams commands
+*/
+
+func newWorkspaceTeamRootCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "team",
+		Short: "Manage Workspace Team resources",
+		Long:  "Teams can be added or removed from Workspaces",
+	}
+	cmd.AddCommand(
+		newWorkspaceTeamAddCmd(out),
+		newWorkspaceTeamUpdateCmd(out),
+		newWorkspaceTeamRmCmd(out),
+		newWorkspaceTeamListCmd(out),
+	)
+	return cmd
+}
+
+func newWorkspaceTeamAddCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "add TEAM",
```

Comment:
New standards: `add` becomes `create`

Reply:
Not sure if `create` fits here since the team already exists outside a workspace and we are adding it to an existing workspace. We are still using `add` for adding users to the workspace. Do we plan to rewrite that as well?

Reply:
UPDATE: see my other comment, for already existing resources we want to use `add` as you wrote.

see [standardization document](https://www.notion.so/astronomerio/CLI-Convergence-Plan-566d2a249c3c4cf9b75e792514ad8023#4d68e54c1514460a9d3050a91ada6739)